torch/nn/_functions/thnn/activation.py

```diff
@@ -95,7 +98,47 @@ def backward(self, grad_output):
             self.lower,
             self.upper,
             self.train,
-            self.inplace
+            False
+        )
+        return grad_input
+
+
+class SELU(InplaceFunction):
+
+    def __init__(self, inplace=False):
+        super(SELU, self).__init__(inplace)
+        self.alpha = 1.6732632423543772848170429916717
+        self.scale = 1.0507009873554804934193349852946
```

Comment:
I didn't mean that they can be user-specified, but that they don't need to be set in `__init__`. You can put the `alpha = ...` in the class body, and it makes `__init__` completely unnecessary in this case